### DISCUSSION TITLE
Reflectance %

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1159] [Q&A] [answered]
I am hoping to normalize the vertical axis for SDs to use reflectance %. Is there a way to do this with Colour? Any suggestions would be welcome and appreciated. -J

**Comment by thomasmansencal** [comment]
Hi @blue-j,

The computed reflectance should already be in percentages. Can you please clarify what you need to do?

Cheers,

Thomas

**Comment by blue-j** [comment]
Thank you as always Thomas for your attention and time.  I am trying to normalize the vertical axis to reflectance percentage so I can stack multiple SDs atop each other, compare, etc.  Something like what Robin Myers has done in his excellent SpectraShop:

![Screen Shot 2023-05-30 at 6 42 52 PM](https://github.com/colour-science/colour/assets/3990142/7a5b0858-61c1-4074-953f-875e7f6b5eca)

**Reply by thomasmansencal** [reply, marked as answer]
```python
import colour

colour.plotting.plot_multi_sds([sd * 100 for sd in  colour.SDS_COLOURCHECKERS["ColorChecker N Ohta"].values()], legend=False, y_label="Reflectance %")
```

![image](https://github.com/colour-science/colour/assets/99779/000b85b7-289e-4719-bd56-5e985202cd28)

**Reply by blue-j** [reply]
is there a way to get this kind of output for a single SD?  i just tried a bunch before asking, please forgive!  - J

**Reply by thomasmansencal** [reply]
Sure, just pass the single sd within a list, i.e. `[sd]`.

**Reply by blue-j** [reply]
i'm gonna push my luck a little here and ask, how do you get fixed axes?  i need 0-100 and 360-830.  i just spent an hour trying to get xlim and ylim passed into the function without joy...  Apologies for pestering you, and much thanks!!

**Reply by thomasmansencal** [reply]
`bounding_box=[400, 700, 0, 100]`
